extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -105,19 +128,19 @@ require([
 				}
 			};
 			uiModal.createComponent(createPageModalConfig, function (modal) {
-				getModalTabContentsHtml(tabToActivate).then(function (tabContentHtml) {
-					var html = navHtml + tabContentHtml;
+				var loading = mustache.render(templates.modalLoadingScreen, {
+					loadingText: $.msg('communitypage-modal-tab-loading'),
+				}), html = navHtml + loading;
 
-					modal.$content
-						.addClass('ContributorsModule ContributorsModuleModal')
-						.html(html)
-						.find(tabToActivate.className).children().addClass('active');
+				modal.$content
+					.addClass('ContributorsModule ContributorsModuleModal')
```

Comment:
In oasis we generally camel case major elements and dash case minor
elements.

On Monday, April 25, 2016, Per Johan Groland <notifications@github.com>
wrote:

> In extensions/wikia/CommunityPage/scripts/ext.communityPage.js
> <https://github.com/Wikia/app/pull/10301#discussion_r61022177>:
>
> >
> > -					modal.$content
> > -						.addClass('ContributorsModule ContributorsModuleModal')
> > -						.html(html)
> > -						.find(tabToActivate.className).children().addClass('active');
> > +				modal.$content
> > +					.addClass('ContributorsModule ContributorsModuleModal')
>
> @lizlux <https://github.com/lizlux> Was there a specific reason for
> making the class names camelCase?
> I think right now, the classes we added for the community page are all
> camelCase.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/10301/files/e99f2996cf415a43fff2807bb39e9b40c6309a37#r61022177>
>


-- 

*Liz Lee*

*Engineer*

e  liz@wikia-inc.com

m 650-867-1913

w  fandom.com

Wikia, Inc.

360 3rd Street, Suite 750

San Francisco, CA 94107


[image: EmailSignature-05.png]

<https://twitter.com/getfandom> <https://www.facebook.com/getfandom/>
<https://www.instagram.com/getfandom/>
<https://www.linkedin.com/company/wikia>

A Wikia, Inc. Property.